components/UserInfoForm.vue

```diff
@@ -261,15 +261,12 @@ export default {
       }
     },
     onEditEmail() {
-      if (this.getUserInfo.isEmailVerified) return;
-      if (this.isEditing) {
-        this.$nextTick(() => this.$refs.inputEmail.$el.focus());
-        return;
-      }
-      this.isEditing = !this.isEditing;
-      if (this.isEditing) {
-        this.$nextTick(() => this.$refs.inputEmail.$el.focus());
-      }
+      if (this.isUserEmailVerified || !this.verifyEmailMission) return;
+      // open mission dialog for verifyEmail directly
+      this.onMissionClick({
+        ...this.verifyEmailMission,
+        isReferral: false,
+      });
```

Comment:
can anything fallback can be used incase `this.verifyEmailMission` is not there?
ie `verificationDialog` or something